internal/doc/draft-user-guide-v1.adoc

```diff
@@ -341,6 +342,46 @@ spec:
       app-monitoring: 'true'
 ----
 
+=== Security Context
+
+A security context is used to control privilege and permission settings for a Pod or a Container. By default, the Runtime Component operator sets several security context parameters for containers as shown below.
```

Comment:
context parameters for containers --> context parameters for application container